mlflow/server/js/src/components/ExperimentViewUtil.js

```diff
@@ -423,6 +423,7 @@ class TreeNode {
       if (visited.has(current.parent.value)) {
         return undefined;
       }
+      visited.add(current.value);
```

Comment:
Small fix, prior to this change we wouldn't actually add the current node to our `visited` dict & so would loop infinitely here if there was a cycle